src/Compiler/Service/ServiceDeclarationLists.fs

```diff
@@ -1107,9 +1089,9 @@ type DeclarationListInfo(declarations: DeclarationListItem[], isForType: bool, i
             items 
             |> List.map (fun x ->
                 match x.Item with
+                | Item.Types (_, TType_app(tcref, _, _) :: _) when isInterfaceTyconRef tcref -> { x with MinorPriority = 1000 + tcref.TyparsNoRange.Length }
```

Comment:
For my understanding, what does this change mean here - why was it needed?

Reply:
That's just a rewrite of the removed pattern below.